tests/unit/test_rat_command.py

```diff
@@ -149,3 +149,31 @@ async def the_command(context: Context):
         await Commands.trigger(ctx)
 
         del Commands._registered_commands[name.casefold()]
+
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize("alias", ['quiet'])
+    async def test_call_command_quiet(self, alias, bot_fx, rat_board_fx, configuration_fx, random_string_fx, monkeypatch):
+        """
+        Verifiy that the !quiet command returns useful information
+        """
+        trigger_alias = f"{configuration_fx.commands.prefix}{alias}"
+        print(f"Triggering alias: {trigger_alias}")
+        bot_fx.board = rat_board_fx
+        ctx = await Context.from_message(bot_fx, "#unittest", "some_rat", trigger_alias)
```

Comment:
[error] please use provided logging facilities instead of `print`ing to STDOUT.

```py
from loguru import logger

logger.debug(...)
```

Reply:
Replaced all (my) prints with loguru